src/ts/components/settings/DynamicSettingsPanelItem.ts

```diff
@@ -22,7 +22,11 @@ export interface DynamicSettingsPanelItemConfig extends SettingsPanelItemConfig
   /**
    * The label component or the text for the label.
    */
-  label: LocalizableText | SubtitleSettingsLabel;
+  label: LocalizableText;
+  /**
+   * Optional component to be displayed next to the label. 
+   */
+  labelComponent?: Component<ComponentConfig>;
```

Comment:
Can we a bit more clear where this additional component will be used? As I understand it, this component is only used in the sub-page in the title. Maybe we could name it accordingly.

```suggestion
  /**
   * Optional component to be displayed next to the label. 
   */
  backNavigationRightTitleComponent?: Component<ComponentConfig>;
```

_Please challenge the naming suggestion if you have anything better_

Reply:
Good point and I was thinking about a better naming for this but then decided to keep the naming in line with the existing naming.

How do you like?
- `additionalNavigationComponent`
- `additionalNavigationRightComponent`

Reply:
I'd encode the fact that it is only used within the back navigation 🤔 `backNavigationRightComponent` maybe?

Reply:
Done: https://github.com/bitmovin/bitmovin-player-ui/pull/696/commits/7c91cc1b1fd86f4e10b8edd8c8ecf8661505ae00